to_do_robot.rb

```diff
@@ -1 +1,109 @@
-puts "Hello!"
+require_relative 'to_do_item_class.rb'
+require_relative 'to_do_list_class.rb'
+puts "Welcome to your To-Do list!"
+
+class MakeToDo
+  def initialize
+    puts "Ready to create your to-do list?(Yes or No?)"
+    answer = gets.chomp.downcase
+
+    case answer
+    when 'yes'
+      puts "What would you like to name your ToDo list?"
+      name = gets.chomp.capitalize
+      @list = List.new(name)
+      puts "Your ToDo list #{list.name} was created!"
+
+    when 'no'
+      puts "Okay, see you later!"
+    else
+      puts "I'm sorry, I didn't understand you, please enter 'Yes' or 'No'"
+    end
+  end
+  def view
+    list.view_list
+  end
+
+  def list
+    @list ||= List.new #if @list is not nil then create a new list and assign it to the list var
+  end
+
+  def add
+    puts "Type the name of one of the items you'd like to do today."
+    new_item = gets.chomp
+
+    if list[new_item.to_sym].nil?
```

Comment:
Because you're not symbolizing the value in your `add_item` method on list, I think this check will always be `nil`, but I would have to play with it to see if that's true.